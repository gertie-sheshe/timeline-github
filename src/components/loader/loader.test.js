import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Loader from './loader';

afterEach(cleanup);

it('Matches Loader Snapshot', () => { 
    const { asFragment } = render(<Loader />);

    expect(asFragment(<Loader />)).toMatchSnapshot();
});
