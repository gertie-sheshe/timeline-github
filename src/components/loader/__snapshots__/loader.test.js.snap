// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches Loader Snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="loader"
    />
  </div>
</DocumentFragment>
`;
